Makefile

```diff
@@ -15,28 +15,17 @@ CLIDSTFONT=$(CLIDST)/node_modules/font-awesome
 default:
 	@echo "Support commands:\ncli api comp copycli copyapi copycomp updateapispec"
 
+
 # Build kubectl docs
 cleancli:
-	sudo rm -f main
 	sudo rm -rf $(shell pwd)/gen-kubectldocs/generators/includes
-	sudo rm -rf $(shell pwd)/gen-kubectldocs/generators/build
-	sudo rm -rf $(shell pwd)/gen-kubectldocs/generators/manifest.json
 
 cli: cleancli
+	# go mod init github.com/generators
```

Comment:
I wonder if the output is something like what we now get from line 43.

Reply:
Yes, but with different `front matter`, heading changes, and integrated sub-commands.